libcxx/utils/ci/run-buildbot-container

```diff
@@ -26,6 +26,6 @@ if [[ ! -d "${MONOREPO_ROOT}/libcxx/utils/ci" ]]; then
     echo "Was unable to find the root of the LLVM monorepo; are you running from within the monorepo?"
     exit 1
 fi
-docker pull ghcr.io/libcxx/libcxx-builder
-docker run -it --volume "${MONOREPO_ROOT}:/llvm" --workdir "/llvm" --cap-add=SYS_PTRACE ghcr.io/libcxx/libcxx-builder \
+docker pull ghcr.io/libcxx/buildkite-builder
```

Comment:
We should change this to the actions-builder image instead, the buildkite builder is increasingly unmaintained and unused.

Reply:
Also, thank you for taking the time to do this.

Reply:
I was wondering whether the buildkite builder was still used. This is the "default" in the Dockerfile's documentation too. @EricWF @ldionne Should we the Buildkite support and documentation from the Dockerfile in a separate patch?

Reply:
I think the `buildkite-builder` image is still used by the `android-buildkite-builder` image, but that's it. So we could refactor it out and remove all mentions of it. Then there would be only the following images
- `actions-builder`
- `android-buildkite-builder`

We could even explore running the Android CI on GitHub Actions at some point if we wanted.

Reply:
I've looked into moving the android builders to github actions prior to January. The android builders need the ability to run QEMU, which requires special permissions and configuration. While those configurations are available on Google Cloud Compute, they're not yet availble in Kubernetes.

We could still probably move them, but they wouldn't be able to scale as aggressively & in sync with the other bots. Not really a huge issue though.